presto-main/docs/t-digest.md

```diff
@@ -0,0 +1,33 @@
+# T-Digest
+
+## Background
+
+The implementation of t-digest in this library is based on [t-digest [Dunning '19]](https://arxiv.org/abs/1902.04023). It has been modified to support Slice serialization.
+
+## Format
+
+_Unless otherwise noted, all values are little-endian._
+
+### T-Digest Layout
+
+![T-Digest Layout](tdigest.png)
```

Comment:
Do we really need to update png files? Or we can just strip them out

Reply:
Having the image makes it a lot easier to understand and is consistent with QuantileDigest documentation, so I think we should keep them.